src/adapters/sqlite/encodeSchema/test.js

```diff
@@ -52,18 +51,84 @@ describe('encodeSchema', () => {
           unsafeSql: (sql) => sql.replace(/create table "tasks" [^)]+\)/, '$& without rowid'),
         }),
       ],
-      unsafeSql: (sql) => `create blabla;${sql}`,
+      unsafeSql: (sql, kind) => {
+        if (kind !== 'setup') {
+          throw new Error('expected different unsafeSql kind')
+        }
+        return `create blabla;${sql}`
+      },
     })
 
-    const expectedSchema =
-      'create blabla;' +
-      expectedCommonSchema +
-      'create table "tasks" ("id" primary key, "_changed", "_status", "author_id") without rowid;' +
-      'create index "tasks_author_id" on "tasks" ("author_id");' +
-      'create index "tasks__status" on "tasks" ("_status");'
+    expect(encodeSchema(testSchema2)).toBe(
+      '' +
+        'create blabla;' +
+        expectedCommonSchema +
+        'create table "tasks" ("id" primary key, "_changed", "_status", "author_id") without rowid;' +
+        'create index "tasks_author_id" on "tasks" ("author_id");' +
+        'create index "tasks__status" on "tasks" ("_status");',
+    )
+  })
+})
 
-    expect(encodeSchema(testSchema)).toBe(expectedSchema)
+describe('encodeIndices', () => {
+  it(`encodes creation of indices`, () => {
+    expect(encodeCreateIndices(testSchema)).toBe(
+      '' +
```

Comment:
Why does this need to be prepended to each query?

Reply:
oh it doesn't, it's a no-op to make Prettier format the query in an easier to read way, nothing to see here :)